etl/pipelines/anti_recommendation_retrieval_pipeline.py

```diff
@@ -22,8 +22,9 @@ def __create_query(
         k: DocumentsLimit,
     ) -> ModelQuestion:
         """Return a query for the retrieval algorithm."""
+        record_key_with_spaces = record_key.replace("_", " ")
```

Comment:
Feels like this should be to_ and from_ on RecordKey? Or at least a static helper somewhere else e.g., a `RecordKeys` class.

Reply:
Yes, I have created a `RecordKeys` class, with methods to handle changes to `record_keys`.